src/macros.rs

```diff
@@ -143,7 +143,7 @@ fn test_vertex_attr_array() {
 /// It ensure the word alignment as well as the magic number.
 #[macro_export]
 macro_rules! include_spirv {
-    ($path:literal) => {
+    ($path:expr) => {
```

Comment:
_Strictly_ speaking, this should probably be `tt*` and we just dump everything inside the arguments to include_bytes, but that feels like way too much work.

Reply:
I made the change.